src/BuildScriptGenerator/PythonVersions.cs

```diff
@@ -9,6 +9,7 @@ public static class PythonVersions
         public const string Python27Version = "2.7.17";
         public const string Python36Version = "3.6.10";
         public const string Python37Version = "3.7.7";
-        public const string Python38Version = "3.8.2";
+        public const string Python38Version = "3.8.3";
+        public const string Python39Version = "3.9.0a6";
```

Comment:
b1?

Reply:
I would wait for b1 little bit ... a6 got released a while ago, b1 yesterday ... so it's more safe to have a6 release for now .. we will release b1 a bit later

Reply:
updated